### DISCUSSION TITLE
Ability to close or mark "Idea" Discussions as implemented?

### DISCUSSION THREAD

**Discussion by mattblovell** [community/community#3192] [Discussions]
Is there any ability to "close" an Idea Discussion, marking it as either implemented or with some other resolution?  Alternatively, can an Idea be closed out once a corresponding "enhancement" Issue is linked?

Presently, with a fairly new Discussion board, I'm seeing the Idea threads just pile up.

**Comment by janpio** [comment]
Related to https://github.com/github/feedback/discussions/3097
